src/modules/notifications/notifications.controller.ts

```diff
@@ -0,0 +1,46 @@
+import { Controller, Get, Req, Query, NotFoundException } from '@nestjs/common';
+import { NotificationsService } from './notifications.service';
+import { UserPayload } from '../user/interfaces/user-payload.interface';
+import { ApiBearerAuth, ApiTags } from '@nestjs/swagger';
+
+@ApiBearerAuth()
+@ApiTags('Notifications')
+@Controller('notifications')
+export class NotificationsController {
+  constructor(private readonly notificationsService: NotificationsService) {}
+
+  @Get()
```

Comment:
Add swagger doc for Ok and error responses. Check other controller files for how to do it.